lib/api_entity.rb

```diff
@@ -72,11 +81,9 @@ def initialize(attributes = {})
   def attributes=(attributes = {})
     @attributes = attributes
 
-    if attributes.present?
-      attributes.each do |name, value|
-        if respond_to?(:"#{name}=")
-          send(:"#{name}=", value.is_a?(String) && value == 'null' ? nil : value)
-        end
+    attributes.each do |name, value|
+      if respond_to?(:"#{name}=")
+        send(:"#{name}=", value)
```

Comment:
`public_send` ? In theory `respond_to?` should only return true for public methods anyway but if its getting updated might be worth correcting

Reply:
Yeah, good shout :)